.github/workflows/automation-ci.yml

```diff
@@ -0,0 +1,183 @@
+          REACT_APP_ENABLE_CARD_SUPPORT=true
+          REACT_APP_ENABLE_TEAMS=true
+          REACT_APP_ENABLE_FHIR_PATIENTS=true
+          REACT_APP_ENABLE_TEAMS_ASSIGNMENT_MODULE=true
+          REACT_APP_ENABLE_FORM_CONFIGURATION=true
+          REACT_APP_ENABLE_FHIR_USER_MANAGEMENT=true
+          REACT_APP_LANGUAGE_CODE=en
+          REACT_APP_PROJECT_CODE=core
+          REACT_APP_FHIR_API_BASE_URL=https://fhir.labs.smartregister.org/fhir
+          REACT_APP_ENABLE_FHIR_GROUP=false
+          REACT_APP_ENABLE_FHIR_PRACTITIONER_ROLE=true
+          REACT_APP_PAGINATION_SIZE=1000
+          REACT_APP_ENABLE_SERVER_SETTINGS=false
+          REACT_APP_OPENSRP_WEB_VERSION=$npm_package_version
+          REACT_APP_DEPLOYMENT_ENVIRONMENT=production
+          REACT_APP_ENABLE_FHIR_ORGANIZATION=true
+          REACT_APP_ENABLE_FHIR_TEAMS=true
+          REACT_APP_FHIR_ROOT_LOCATION_IDENTIFIER=eff94f33-c356-4634-8795-d52340706ba9
+          REACT_APP_FHIR_PATIENT_SORT_FIELDS=-_lastUpdated
+          REACT_APP_FHIR_PATIENT_BUNDLE_SIZE=5000
+          REACT_APP_ENABLE_FHIR_HEALTHCARE_SERVICES=false
+          REACT_APP_ENABLE_USER_MANAGEMENT=true
+          REACT_APP_ENABLE_FHIR_COMMODITY=true
+          REACT_APP_ENABLE_QUEST=true
+          EOF
+
+      # cache builds or artifacts, probably cache - TODO
+      - name: Build Application
+        run: |
+          cd web
+          yarn install --frozen-lockfile
+          yarn lerna:prepublish
+
+      - name: Get absolute path for build files
+        run: |
+          echo "REACT_BUILD_FOLDER=$(pwd)/web/app/build" >> $GITHUB_ENV
+
+      - name: Build express application
+        run: |
+          cd express
+          yarn install --frozen-lockfile
+          yarn build
+          yarn start > /dev/null 2>&1 &
+        env:
+          NODE_ENV: development # so that we can insall devDepenencies required for build
+          EXPRESS_OPENSRP_ACCESS_TOKEN_URL: https://keycloak-stage.smartregister.org/auth/realms/FHIR_Android/protocol/openid-connect/token
+          EXPRESS_OPENSRP_AUTHORIZATION_URL: https://keycloak-stage.smartregister.org/auth/realms/FHIR_Android/protocol/openid-connect/auth
+          EXPRESS_KEYCLOAK_LOGOUT_URL: https://keycloak-stage.smartregister.org/auth/realms/FHIR_Android/protocol/openid-connect/logout
+          EXPRESS_OPENSRP_USER_URL: https://keycloak-stage.smartregister.org/auth/realms/FHIR_Android/protocol/openid-connect/userinfo
+          EXPRESS_OPENSRP_CLIENT_ID: fhir-core-client
+          EXPRESS_OPENSRP_CLIENT_SECRET: ${{ secrets.EXPRESS_OPENSRP_CLIENT_SECRET }}
+          EXPRESS_ALLOW_TOKEN_RENEWAL: true
+          EXPRESS_REACT_BUILD_PATH: ${{ env.REACT_BUILD_FOLDER }}
+          EXPRESS_CONTENT_SECURITY_POLICY_CONFIG: "{\"default-src\":[\"'self'\",\"smartregister.org\",\"https://github.com/opensrp/\"],\"connect-src\":[\"'self'\",\"*.smartregister.org\"],\"img-src\":[\"https://github.com/opensrp/\"],\"script-src\":[\"'self'\",\"'unsafe-inline'\"]}"
```

Comment:
@p-netm we can have the new csp approach [here](https://github.com/opensrp/web/commit/7191b5bf8c1b18114a0c8f6f0499b18df3679550) and remove the unsafe-inline directive